### DISCUSSION TITLE
How far away is this library from version 3?

### DISCUSSION THREAD

**Discussion by javierbegarie** [the-guild-org/apollo-angular#1665] [Q&A]
I'm asking because I had to stop the migration from v1 to v2 due to the lack of lazy loading capabilities in the new version.  We do microfrontends and not all the teams use GraphQL, so the workaround of declaring the provider at the root app module is not suitable for us. 

But I've read in the ticket issues that it's the plan to recover the ApolloModule in version 3. 

Anyway, I've never collaborated with an open source project, and sorry if the answer to my next sentence is quite obvious but if you need help I can collaborate with that.

Thanks in advance.

**Comment by kamilkisiela** [comment]
@javierbegarie You can use `apollo-angular@next`. It's almost ready and I just need to push few fixes (mostly typescript related) there.
